spec/support/publishing_api_helpers.rb

```diff
@@ -1,11 +1,12 @@
 module PublishingApiHelpers
   def write_payload(document)
-    document.delete("last_edited_at")
-    document.delete("publication_state")
-    document.delete("first_published_at")
-    document.delete("public_updated_at")
-    document.delete("state_history")
-    document
+    copy = FactoryGirl.create(document["document_type"], document)
```

Comment:
As discussed, ignore the comment above. You're passing `document` in so it should preserve changes to the document's attributes made in the test.